### PR TITLE
run_worker.sh updates

### PR DESCRIPTION
**Pull request by jpopelka**
*  Exit on failed commands or undefined variables (kudos to Hunor for repeatedly bringing this to my attention)
*  [Simplify setting of default values](https://stackoverflow.com/a/28085062/2768738)

### CODE REVIEW

File: files/run_worker.sh [hunk 1]
```diff
@@ -12,40 +14,34 @@ source /usr/bin/setup_env_in_openshift.sh
...
+if [[ "${CELERY_COMMAND:=worker}" == "beat" ]]; then
     # when using the database backend, celery beat must be running for the results to be expired.
     # https://docs.celeryq.dev/en/stable/userguide/periodic-tasks.html#starting-the-scheduler
     exec celery --app="${APP}" beat --loglevel="${LOGLEVEL:-DEBUG}" --pidfile=/tmp/celerybeat.pid --schedule=/tmp/celerybeat-schedule
 
 elif [[ "${CELERY_COMMAND}" == "worker" ]]; then
     # define queues to serve
-    DEFAULT_QUEUES="short-running,long-running"
-    QUEUES="${QUEUES:-$DEFAULT_QUEUES}"
+    : "${QUEUES:=short-running,long-running}"
```
Comment (TomasTomecek) [inline files/run_worker.sh:32]: first time seeing this, wow

File: files/run_worker.sh [hunk 2]
```diff
@@ -12,40 +14,34 @@ source /usr/bin/setup_env_in_openshift.sh
...
 grep -q pkgs.fedoraproject.org known_hosts || ssh-keyscan pkgs.fedoraproject.org >>known_hosts
-popd || exit
+popd
 
-DEFAULT_CELERY_COMMAND="worker"
 # Whether to run Celery worker or beat (task scheduler)
-CELERY_COMMAND="${CELERY_COMMAND:-$DEFAULT_CELERY_COMMAND}"
-
-if [[ "${CELERY_COMMAND}" == "beat" ]]; then
+if [[ "${CELERY_COMMAND:=worker}" == "beat" ]]; then
```
Comment (TomasTomecek) [inline files/run_worker.sh:25]: had to read definition of `${foo:=bar}` to understand this:

```
${parameter:=word}
If parameter is unset or null, the expansion of word is assigned to parameter. The value of parameter is then substituted. Positional parameters and special parameters may not be assigned to in this way.
```

https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

+1

**Review by TomasTomecek — APPROVED** [review]
lgtm, more bash magic :)

**Comment by jpopelka** [comment]
can't reproduce the pre-commit error, merging manually

**Pull request merged**